renovate.json5

```diff
@@ -12,5 +12,27 @@
         executionMode: "branch", // Only run once.
       },
     },
+    {
+      matchManagers: ["gomod"],
+      matchPackageNames: ["github.com/pulumi/pulumi/pkg/v3", "github.com/pulumi/pulumi/sdk/v3"],
+      postUpgradeTasks: {
+        commands: ["make renovate"],
+        executionMode: "branch", // Only run once. 
+      },
+      // automatically bump these, don't require manual trigger
+      dependencyDashboardApproval: false,
+    },
```

Comment:
It's not clear to me (yet) how this is going to interact with the `pulumi` CLI version. I suspect this may not be enough.